src/Microsoft.Health.Dicom.Core/Features/Retrieve/DicomFileExtensions.cs

```diff
@@ -1,4 +1,4 @@
-﻿// -------------------------------------------------------------------------------------------------
```

Comment:
Revert?

Reply:
compiler is throwing style errors without this

Reply:
How come? This file is not changed and it is building in main